ui/app/styles/core/charts.scss

```diff
@@ -204,22 +202,22 @@ h2.chart-title {
 
 p.chart-description {
   color: $ui-gray-700;
-  font-size: $body-size;
```

Comment:
I have a ticket to address the replacement of this var. It was originally 14px, but in some cases it's replaced with size-8 (an equivalent REM value), which causes issues in some cases.